contrib/mod_ifsession.c

```diff
@@ -1222,6 +1179,8 @@ MODRET ifsess_post_user(cmd_rec *cmd) {
 
 static void ifsess_chroot_ev(const void *event_data, void *user_data) {
   ifsess_home_dir = (const char *) event_data;
+
+  ifsess_sess_process_displayfiles();
```

Comment:
This works because at the time the `core.chroot` event is published, authentication (and thus reading of the `AuthUserFile`, `AuthGroupFile` files) will have already occurred, and the necessary applicable user/group lookups will have occurred; we don't need to do those lookups again ourselves.

Previously, the `mod_ifsession` _would_ do those lookups as part of the `PRE_CMD` phase of the `PASS` command, which happens _before_ the core `mod_auth` module does _its_ lookups -- resulting in redundant user/group membership lookups, redundant `AuthUserFile`, `AuthGroupFile` scans.